compiler/next/include/chpl/uast/ASTClassesList.h

```diff
@@ -67,7 +67,9 @@ AST_BEGIN_SUBCLASSES(Expression)       // old AST: Expr
   //AST_NODE(Import)                     // old AST: ImportStmt
   AST_NODE(New)
   //AST_NODE(Require)                    //
-  //AST_NODE(Use)                        // old AST: UseStmt
+  AST_NODE(As)                         //
+  AST_NODE(Use)                        // old AST: UseStmt
+  AST_NODE(UseClause)                  //
```

Comment:
The chunks seem to usually be arranged alphabetically, but this breaks that.  Does this imply that Use and related nodes should end up in their own chunk?

Reply:
Nope, that's just an error on my part.

Reply:
Actually @mppf do we want to make an effort to arrange everything alphabetically? I'm actually kind of worried that the deep levels of nesting for decls will make things hard to follow (perhaps unfounded).

Reply:
The idea was to have the ones that aren't parent classes/groups in sorted order, followed by the parent classes/groups. The file is mostly already organized in this manner but there are probably a few other drifts away from it.

Reply:
I sorted everything but the abstract class "chunks".